config.def.h

```diff
@@ -451,7 +452,7 @@ static Button buttons[] = {
 	{ ClkStatusText,        MODKEY,         Button4,        spawn,          {.v = upbright } },
 	{ ClkStatusText,        MODKEY,         Button5,        spawn,          {.v = downbright } },
 	{ ClkRootWin,           MODKEY,         Button3,        spawn,          {.v = notifycmd } },
-	{ ClkRootWin,           0,              Button1,        spawn,          {.v = panther } },
+	// { ClkRootWin,           0,              Button1,        spawn,          {.v = panther } },
```

Comment:
If you're gonna remove it, remove it. Don't leave the comment a dangling zombie for all eternity :wink:

Reply:
Good point :wink:  done.